src/utp_stream.cpp

```diff
@@ -3147,7 +3147,7 @@ bool utp_socket_impl::incoming_packet(span<std::uint8_t const> buf
 				m_connect_handler = false;
 				utp_stream::on_connect(m_userdata, m_error, false);
 			}
-			// fall through
+			BOOST_FALLTHROUGH;
```

Comment:
ah, it's in RC_1_1, but I haven't merged that into master for a while. feel free to change it i master and I'll merge later